pilot/cmd/pilot-agent/main.go

```diff
@@ -62,7 +62,7 @@ const trustworthyJWTPath = "/var/run/secrets/tokens/istio-token"
 var (
 	role          = &model.Proxy{}
 	proxyIP       string
-	registry      serviceregistry.ServiceRegistry
+	registry      serviceregistry.ProviderID
```

Comment:
RegistryID

Reply:
Not sure I agree. `RegistryID` implies that it's unique to the instance. `ProviderID` implies that it's unique to the provider. The way it's used to day is as a provider, so I'm trying to make that clear.

Reply:
oh nm ... I thought you were referring to the use of `ProviderID`. Fixed.